lib/rouge/lexers/hocon.rb

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'json.rb'
+
+    class HOCON < JSON
+      title 'HOCON'
+      desc "Human-Optimized Object Configuration Notation (https://github.com/lightbend/config)"
+      tag 'hocon'
+      filenames '*.hocon'
+
+      prepend :root do
+        # Comments
+        rule %r(//.*?$), Comment::Single
+        rule %r(#.*?$), Comment::Single
+
+        # Interpolation
+        rule %r/[$][{][?]?/, Literal::String::Interpol, :interpolation
+
+        # Strings
+        rule %r/"""/, Literal::String::Double, :multiline_string
+
+        # Keywords
+        rule %r/\b(?:include|url|file|classpath)\b/, Keyword
+
+        # Symbols
+        rule %r/[()={},:\[\]]/, Punctuation
```

Comment:
Are these values not picked up by the JSON lexer?

Reply:
I've removed the characters that were already handled by the punctuation rule in the JSON lexer. I think the remaining characters are still necessary.

Reply:
Yep, looks good.